ontology.ttl

```diff
@@ -255,8 +255,9 @@ owl:versionIRI rdf:type owl:AnnotationProperty .
          rdfs:subClassOf :WorkflowParameter ,
                          [ rdf:type owl:Restriction ;
                            owl:onProperty :value ;
-                           owl:allValuesFrom xsd:boolean
-                         ] .
+                           owl:allValuesFrom xsd:boolean, xsd:bool
```

Comment:
I'm not sure xsd:bool exists. I can't find it anywhere, only xsd:boolean

Reply:
`bool` is the data-type of Python, same for `int` instead of `integer`

Reply:
btu anyway, this is an issue that should be handled at another level, inside `nb2workflow`